src/components/SpaceShip/SpaceShip.tsx

```diff
@@ -1,9 +1,13 @@
-export const SpaceShip = () => {
+interface SpaceShipProps {
+  hasBanner?: boolean;
+}
+
+export const SpaceShip = ({ hasBanner }: SpaceShipProps) => {
   return (
     <div className="spaceship-wrapper">
       <div className="spaceship-inner">
         <svg
-          className="spaceship spaceship--with-banner"
+          className={`spaceship ${hasBanner ? ' spaceship--with-banner' : ''}`}
```

Comment:
This `--with-banner` class is so the spaceship isn't covered up when the banner is present in the content.